.github/workflows/ci.yaml

```
@@ -0,0 +1,17 @@
+name: CI
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  lint-all:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
```

```suggestion
          node-version: 22
```

Let's use newer node version (most of our repositories are updated to v22).